### PR TITLE
Private/gokay/impress accessibility definitions

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/Control.NotebookbarImpress.js [hunk 1]
```diff
@@ -137,140 +153,138 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
...
-		if (hasSaveAs) {
-			if (hasGroupedSaveAs) {
-				content.push({
-					'id': 'saveas',
+			}: {},
+			(!this._map['wopi'].UserCanNotWriteRelative) ?
+			(
+				(window.uiDefaults && window.uiDefaults.saveAsMode === 'group') ?
+				{
+					'id': 'file-saveas',
```
Comment (pedropintosilva) [inline browser/src/control/Control.NotebookbarImpress.js:174]: the button is coming out without icon please add icon alias:
```diff
diff --git a/browser/src/control/Control.JSDialogBuilder.js b/browser/src/control/Control.JSDialogBuilder.js
index 6abb6e6627..d45d330458 100644
--- a/browser/src/control/Control.JSDialogBuilder.js
+++ b/browser/src/control/Control.JSDialogBuilder.js
@@ -2320,6 +2320,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		}
 
 		var iconURLAliases = {
+			'file-saveas': 'saveas',
 			'home-search': 'recsearch',
 			'addmb-menu': 'ok',
 			'closetablet': 'view',
```

Also the save as should be a dropdown but it looks like this now:
image

File: browser/src/control/Control.NotebookbarImpress.js [hunk 2]
```diff
@@ -1020,14 +1185,18 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
...
 				'id': 'HyperlinkDialog',
+				'class': 'unoHyperlinkDialog',
 				'type': 'bigcustomtoolitem',
 				'text': _UNO('.uno:HyperlinkDialog'),
-				'command': 'hyperlinkdialog'
+				'command': 'hyperlinkdialog',
+				'accessibility': { focusBack: true, combination: 'IL', de: null }
 			},
 			(this._map['wopi'].EnableRemoteLinkPicker) ? {
+				'id': 'insert-insert-smart-picker',
```
Comment (pedropintosilva) [inline browser/src/control/Control.NotebookbarImpress.js:1195]: please add 
```js
'class': 'unoremotelink'.
```

File: browser/src/control/Control.NotebookbarImpress.js [hunk 3]
```diff
@@ -420,7 +455,8 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				'id': 'toggleuimode',
 				'type': 'bigmenubartoolitem',
 				'text': _('Compact view'),
-				'command': _('Toggle UI Mode')
+				'command': _('Toggle UI Mode'),
+				'accessibility': { focusBack: true, combination: 'TU', de: null }
```
Comment (pedropintosilva) [inline browser/src/control/Control.NotebookbarImpress.js:459]: please add:
```js
'class': 'unotoggleuimode',
```

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/src/dom/NotebookbarAccessibility.js [hunk 4]
```diff
@@ -27,15 +27,27 @@ var NotebookbarAccessibility = function() {
...
-
-		return infoBox;
+		var visible = anchorElement.id.replace('-button', '');
+		visible = document.getElementById(visible);
+
+		if (visible)
+			visible = visible.style.display !== 'none';
+
+		if (visible) {
+			console.log('display: ' + anchorElement.style.display);
```
Comment (eszkadev) [inline browser/src/dom/NotebookbarAccessibility.js:37]: console log, to delete I think?

File: browser/src/control/Control.NotebookbarImpress.js [hunk 5]
```diff
@@ -58,63 +63,74 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
...
-				'context': 'MasterPage'
+				'context': 'MasterPage',
+				'accessibility': { focusBack: false, combination: 'M', de: null }
 			},
 			{
-				'text': _('View'),
 				'id': 'View-tab-label',
+				'text': _('View'),
 				'name': 'View',
+				'accessibility': { focusBack: false, combination: 'W', de: null }
```
Comment (eszkadev) [inline browser/src/control/Control.NotebookbarImpress.js:127]: what is de? why we have to assign null there?

**Review by eszkadev — APPROVED** [review]

**Pull request merged**